packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -1169,13 +1169,15 @@ export class GarbageCollector implements IGarbageCollector {
     }
 
     /**
-     * Logs an event if a node is inactive and is used.
+     * Called when a node is used. If the node is inactive, queue up an event that will be logged next time GC runs.
      */
-    private logIfInactive(
-        eventType: "Changed" | "Loaded" | "Revived",
+    private inactiveNodeUsed(
```

Comment:
`logInactiveNodeUsed`? or `raiseInactiveNodeUsedEvent` or `raiseEventInactiveNodeUsed`?

Reply:
I removed the term log because it may or may not log. The logging happens in logPendingEvents which has the "term" log.